setup.cfg

```diff
@@ -0,0 +1,47 @@
+[coverage:report]
+precision = 2
+exclude_lines =
+    @abc.abstractmethod
+    @abc.abstractproperty
+    NotImplementedError
+    pragma: no cover
+    __repr__
+    __str__
+
+[coverage:run]
+branch = True
+source =
+    src
+    tests
+omit =
+    tests/utils.py
+
+[isort]
+atomic = true
+default_section = FIRSTPARTY
+include_trailing_comma = true
+indent = '    '
+known_first_party = airflow_declarative
+known_third_party =
+line_length = 79
+lines_after_imports = 2
+multi_line_output = 3
+order_by_type = true
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+
+[tool:pytest]
+python_files = tests/**/*.py tests/*.py
```

Comment:
Can we not have this option? Or if we can't, it need to be written as a list.

Reply:
Otherwise pytest scans the whole working dir searching for tests, which takes forever. I'm not sure where exactly does it stuck, probably at the `.tox` dir, or maybe at `venv`. I'll see if I can get rid of this line.

Reply:
I just removed it and it worked without any issues. Apparently the freeze I was talking about was caused by something else while I was getting everything together.